about/moveit-status/index.markdown

```diff
@@ -5,15 +5,22 @@ date: 2013-12-08 05:22:09+00:00
 layout: page
 slug: moveit-status
 title: Status
-wordpress_id: 164
 ---
 
-This page highlights the status of MoveIt! components - it indicates how stable individual components are and how well they have been tested. It also indicates how much support users can expect for different components and robots. (See legend below for description of the codes.)
+# MoveIt! Status
 
+## Roadmap
 
-### Current STATUS (Components)
+Upcoming milestones as of September 2016:
 
+- [First release of Kinetic](https://github.com/ros-planning/moveit/issues/18)
+- [New release of Indigo](https://github.com/ros-planning/moveit/issues/100)
```

Comment:
I'm not sure. I was inspired by the "Project Status" section of the homepage of http://gazebosim.org/. To combat the transient nature I added a date above it in case it falls behind. But overall talking about major releases isn't that often of a change, and relevant to new users who might not know to search deep within our issue tracker for those tickets. I like it.